Multisig/Logic/PrivateKeyController.swift

```diff
@@ -26,8 +28,41 @@ class PrivateKeyController {
         }
     }
 
+    static func importKey(session: Session, installedWallet: InstalledWallet?) -> Bool {
+        do {
+            try KeyInfo.import(session: session, installedWallet: installedWallet)
+
+            Tracker.shared.setNumKeysImported(KeyInfo.count)
+            NotificationCenter.default.post(name: .ownerKeyImported, object: nil)
+            return true
+        } catch {
+            if let err = error as? GSError.CouldNotImportOwnerKeyWithSameAddress {
+                App.shared.snackbar.show(error: err)
+            } else {
+                let err = GSError.error(description: "Failed to add WalletConnect owner", error: error)
+                App.shared.snackbar.show(error: err)
+            }
+            return false
+        }
+    }
+
+    static func updateKey(session: Session, installedWallet: InstalledWallet?) -> Bool {
```

Comment:
interesting what is the difference between usage of this and `importKey` function

Reply:
comment needed

Reply:
added